src/Main.java

```diff
@@ -1,5 +1,70 @@
 public class Main {
+    private static final Employee[] empl = new Employee[10];
+
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        empl[0] = new Employee("ФИО1", "Отдел1", 25_000);
+        empl[1] = new Employee("ФИО2", "Отдел2", 26_000);
+        empl[2] = new Employee("ФИО3", "Отдел1", 27_000);
+        empl[3] = new Employee("ФИО4", "Отдел2", 28_000);
+        empl[4] = new Employee("ФИО5", "Отдел3", 29_000);
+        empl[5] = new Employee("ФИО6", "Отдел4", 30_000);
+        empl[6] = new Employee("ФИО7", "Отдел3", 31_000);
+        empl[7] = new Employee("ФИО8", "Отдел4", 32_000);
+        empl[8] = new Employee("ФИО9", "Отдел5", 33_000);
+        empl[9] = new Employee("ФИО10", "Отдел5", 34_000);
+        allEmpl();
+        salaryInMonth();
+        emplMinSalary();
+        emplMaxSalary();
+        emplMidlSalary();
     }
-}
\ No newline at end of file
+// Получить список всех сотрудников.
+    private static void allEmpl() {
+        for (Employee employee : empl) System.out.println(employee);
```

Comment:
```suggestion
        for (Employee employee : empl) {
            System.out.println(employee);
        }
```
фигурные скобки надо всегда писать после if/else/for/while